packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -205,7 +205,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 onInputValueChange={ this.onInputChange }
                 suggestedDisplayValue={ suggestedDisplayValue }
                 aria-activedescendant={ activeDescendant }
-                aria-owns='suggestion-list'
+                aria-owns={ this.state.suggestionsVisible ? 'suggestion-list' : '' }
```

Comment:
Should we use `undefined` rather than blank string?

Reply:
Thanks. Will update!